src/module/BlockBingo.h

```diff
@@ -14,10 +14,11 @@
 class BlockBingo {
  private:
   Controller& controller;
-  // Navigator navigator;
+  Navigator navigator;
   //交点サークルから中点までの距離
   const int targetBrightness;
-  const double length;
+  const double lengthCrossCircleCenter;
```

Comment:
定数なら`constexpr`で良くね？
```suggestion
  static constexpr double lengthCrossCircleCenter = 175.0;
```